src/common/libflux/message.h

```diff
@@ -56,26 +57,41 @@ struct flux_match {
     const char *topic_glob;       /* glob matching topic string (or NULL) */
 };
 
-#define FLUX_MATCH_ANY (struct flux_match){ \
-    .typemask = FLUX_MSGTYPE_ANY, \
-    .matchtag = FLUX_MATCHTAG_NONE, \
-    .topic_glob = NULL, \
+static inline struct flux_match flux_match_create (int typemask,
+                                                   uint32_t matchtag,
+                                                   const char *topic_glob)
+{
+    struct flux_match m = {typemask, matchtag, topic_glob};
```

Comment:
Create/destroy are usually a symmetrical pair, but the `topic_glob` refrence is "stolen" here.   I like @grondo's idea of renaming to `flux_match_init()`.

Reply:
Would you both be good with a rename but the same functionality?  I'd be good with that.  Go with the `_init` and `_free` names?

Reply:
Yup